ui/components/requirements/policy-link.js

```diff
@@ -3,13 +3,19 @@ import React from 'react';
 
 import { Link } from '../../routes';
 
+export function policyLinkTag(policy) {
+  return (
+    <Link route="policies" params={{ id__in: policy.id }}>
+      <a>{policy.title_with_number}</a>
+    </Link>
+  );
+}
```

Comment:
Should this be a component? Components can be functions, but offer a lot of extra React functionality.

Reply:
@cmc333333 This was done to quiet codeclimate. I think if we want, we could turn this into a component, but I would wait to see where this goes.